### PR TITLE
Create Rule22-17.md

### LINKED ISSUE #305: Create Rule22-17.md
Heat Rejection Design Supply Temperature is correct

### PR DESCRIPTION
**Pull request by xingcx**
resolves #305

### CODE REVIEW

File: docs/section22/Rule22-17.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+## Rule Logic:  
+
+- For each heat rejection device in B_RMR: `for tower_b in B_RMR.RulesetModelInstance.heat_rejections:`
+
+  - Get reported design wet-bulb temperature: `wbt_b = tower_b.design_wetbulb_temperature`
+
+  - Get reported approach temperature: `approach_b = tower_b.approach`
+
+  - Get reported design supply temperature: `supply_temperature_b = tower_b.design_supply_temperature`
+
```
Comment (mkarpman) [inline docs/section22/Rule22-17.md:35]: I wonder if this should be FluidLoop.cooling_or_condensing)design_and_control.design_supply_temperature. In general, there seems to be several fields in the HeatRejection data group that are redundant to FluidLoop.DesignAndControl

Comment (xingcx) [reply docs/section22/Rule22-17.md:35]: Updated as per schema v0.0.12.

File: docs/section22/Rule22-17.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  - Get reported design wet-bulb temperature: `wbt_b = tower_b.design_wetbulb_temperature`
+
+  - Get reported approach temperature: `approach_b = tower_b.approach`
+
+  - Get condenser water loop served by heat rejection device: `cw_loop_id_b = tower_b.loop`
+
+    - Get design supply temperature of condenser loop: `supply_temperature_b = get_component_by_id(cw_loop_id_b).cooling_or_condensing_design_and_control.design_supply_temperature`
+
+    **Rule Assertion:**
+
```
Comment (supriyagoel) [inline docs/section22/Rule22-17.md:41]: question: should we be checking if wet bulb is in between 55 and 95 as an applicability check as well?
i know we discussed this but I forget what the outcome was.

Comment (xingcx) [reply docs/section22/Rule22-17.md:41]: I added an applicability check and a note regarding models with multiple heat rejection devices.

**Review by mkarpman — APPROVED** [review]

**Review by supriyagoel — APPROVED** [review]

**Pull request merged**